code/game/machinery/computer/communications.dm

```diff
@@ -273,6 +273,16 @@
 			src.emagged = 0
 			setMenuState(usr,COMM_SCREEN_MAIN)
 
+		if("RestartNanoMob")
+			if(mob_hunt_server)
+				var/loading_msg = pick("Respawning spawns", "Reticulating splines", "Flipping hat",
+									"Capturing all of them", "Fixing minor text issues", "Being the very best",
+									"Nerfing this", "Not communicating with playerbase", "Coding a ripoff in a 2D spaceman game")

```

Comment:
it looks right in dream-maker because github displays tabs differently for some reason, just like i pointed out to taukausanake in their now-outdated comment